Makefile

```diff
@@ -11,7 +11,8 @@ installProd:
 
 developmentTests:
 	cd ${APP_DIRECTORY}; npm run audit:ci;
-	cd ${APP_DIRECTORY}; npm run test; xvfb-run npm run test:e2e:storybook:ci
+	cd ${APP_DIRECTORY}; npm run test;
+	cd ${APP_DIRECTORY}; xvfb-run npm run test:e2e:storybook:ci
```

Comment:
This had to be moved to a new step due to a race condition that caused errors to be ignored on jenkins